templates/gitlab-multi-runner/4/docker-compose.yml.tpl

```diff
@@ -29,18 +29,27 @@ services:
     labels:
       io.rancher.container.start_once: 'true'
 
+  gitlab-runner-config-edit:
```

Comment:
Could you please set network to none for this sidekick??
```
+    network_mode: none
```

Reply:
Nice catch ! Done